jobeet/.gitignore

```diff
@@ -12,3 +12,5 @@
 /src/Acme
 /src/AppBundle
 .idea/
+web/data
+*.php~
```

Comment:
Jaaa tényleg a büdös csicska... már el is felejtettem...nem baj akkor se
kell a githez adni

Juhász Ádám <notifications@github.com> ezt írta (időpont: 2015. ápr. 21.,
Ke 18:31):

> In jobeet/.gitignore
> <https://github.com/JuhaszAdam/JobeetSymfony1/pull/14#discussion_r28796282>
> :
>
> > @@ -12,3 +12,5 @@
> >  /src/Acme
> >  /src/AppBundle
> >  .idea/
> > +web/data
> > +*.php~
>
> nem gedit, arról már rég lelőttem azt a funkciót, a doctrine entitás
> generálása csinálja ezt, de kiszedem akkor
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/JuhaszAdam/JobeetSymfony1/pull/14/files#r28796282>.
>